MANIFEST.in

```diff
@@ -1,5 +0,0 @@
-include versioneer.py
-include xdsl/_version.py
-include requirements.txt requirements-optional.txt
```

Comment:
the fact that requirements-optional has been gone for a while and nothing was flagged is already intereseting. I also don't see how users will make use of the requirements.txt